ql/src/java/org/apache/hadoop/hive/ql/io/orc/WriterImpl.java

```diff
@@ -582,7 +593,7 @@ public boolean hasWriterTimeZone() {
   private abstract static class TreeWriter {
```

Comment:
This patch doesn't actually change the write path. I've done this in HIVE-12055 including adding a new writer version.